index.html

```diff
@@ -259,6 +270,134 @@ <h3>Video Content Hints</h3>
+          <li>Otherwise, choose a value for the setting corresponding to c
+            that satisfies the applied constraints.</li>
+          <li>Remember the value <var>t</var>.
+          <li>In parallel, update the trac with the new settings.</li>
+        </ul>
+      <p>Whenever the "apply constraints" algorithm is subsequently
+        run, the UA MUST choose the remembered value <var>t</var> if it is
+        now a permitted value.
+      </p>
+      <p>When setting a contentHint value of "", all remembered values of
+        <var>t</var> are removed.
+      </p>
+    </section>
+    <section>
+      <h3>Degradation preference when encoding</h3>
+      <p>
+        When encoding video, and some constraint (bandwidth, CPU) prevents
+        encoding at the configured framerate and resolution, the encoder
+        must make a choice on how to modify the encoding parameters.
+      </p>
+      <p>
+        This section defines terms to describe the choice, and an enum
+        that can be used to indicate that choice in an API.
+      </p>
+      <div>
+        <pre class="idl"
+>enum RTCDegradationPreference {
+  "maintain-framerate",
+  "maintain-resolution",
+  "balanced"
+};</pre>
+        <table data-link-for="RTCDegradationPreference" data-dfn-for=
+        "RTCDegradationPreference" class="simple">
+          <tbody>
+            <tr>
+              <th colspan="2"><code><a>RTCDegradationPreference</a></code> Enumeration description</th>
+            </tr>
+            <tr>
+              <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>maintain-framerate</code></dfn></td>
+              <td>
+                <p>Degrade resolution in order to maintain framerate.</p>
+              </td>
+            </tr>
+            <tr>
+              <td><dfn data-idl><code>maintain-resolution</code></dfn></td>
+              <td>
+                <p>Degrade framerate in order to maintain resolution.</p>
+              </td>
+            </tr>
+            <tr>
+              <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>balanced</code></dfn></td>
+              <td>
+                <p>Degrade a balance of framerate and resolution.</p>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      An attribute is defined for RTCRtpSendParameters that allows this
+      to be explicitly indicated for an RTCRtpSender:
+
+      <pre class='idl'>partial dictionary RTCRtpSendParameters {
+        RTCDegradationPreference degradationPreference;
+       };</pre>
+      <section>
+        <h2>Dictionary <code><a>RTCRtpSendParameters</a></code> New Members</h2>
+        <dl>
+          <dt data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>degradationPreference</code></dfn> of type
+            <span class="idlMemberType"><a>RTCDegradationPreference</a></span>,
+            defaulting to <code>"balanced"</code></dt>
```

Comment:
It's a bit odd to say it defaults to "balanced" when the dictionary doesn't have a default value (for good reasons, we want the automatic switching depending on the contentHint to work).

Reply:
Yeah, I missed removing this. Will fix.

Reply:
Fixed.